sdk/javascript/generator/AbstractTypeFormatter.py

```diff
@@ -16,6 +16,10 @@ class AbstractTypeFormatter(ABC):
 	def typename(self):
 		raise NotImplementedError('need to override method')
 
+	@property
+	def is_abstract(self):
+		return False
```

Comment:
At first, I thought this referred to the formatted class....maybe ``is_abstract_type/class``?